### DISCUSSION TITLE
Tailwind CSS Purge is not working

### DISCUSSION THREAD

**Discussion by akashshyamdev** [tailwindlabs/tailwindcss#5538] [Help] [answered]
Hey Guys,

I'm trying to use tailwindcss and webpack. It's working fine but it is not purging the styles properly. For example, I'm not using `flex-grow` anywhere but I still find the utilities in my CSS:
![Screenshot 2021-09-19 at 09 59 25](https://user-images.githubusercontent.com/56759828/133915515-eb76a281-b05f-4c09-b58f-4395ad035720.png)

Here's the repo - https://github.com/akashshyamdev/portfolio-html

I'd appreciate it if you could look at it and help me out! Thanks, looking forward to your reply!

**Comment by simonswiss** [comment, marked as answer]
Hey!

Not exactly sure what happens with your webpack setup, but I was able to get the purging to work correctly by adding `NODE_ENV=production` to the `build` script in your `package.json`.

![image](https://user-images.githubusercontent.com/485747/134122655-cc618aec-f582-487c-8961-565a437cfa8d.png)

The default behaviour for `purge` is to only run when that environment variable is set to `production`, and it seems to do the trick in your project.

That takes your CSS bundle from ~ 1.5Mb to a very nice `8.6kB`, and **under 3kB** Gzipped!

![image](https://user-images.githubusercontent.com/485747/134122969-0ddd02c4-382d-499f-8438-4becd3029739.png)

**Reply by akashshyamdev** [reply]
@simonswiss thanks a lot buddy! I'd love to contribute to tailwind css, are there any beginner friendly issues?
